### DISCUSSION TITLE
Can I use file paths as table names for DuckDB?

### DISCUSSION THREAD

**Discussion by eitsupi** [PRQL/prql#1493] [Q&A] [answered]
DuckDB can accept the path to a Parquet file as the table name.
For example, `FROM 'test/*.parquet'`.
Is there any way to use this in PRQL?
https://duckdb.org/docs/data/parquet.html

It seems that s-string cannot be used for this purpose.

**Comment by max-sixty** [comment, marked as answer]
Sorry for this. 

I opened https://github.com/PRQL/prql/issues/1498 to make this better.

For the moment, this is a workaround:

```elm
from s"select * from 'test/*.parquet'"
```

**Reply by eitsupi** [reply]
Thank you for your answer!
This may be a DuckDB specific issue, but given the popularity of DuckDB it would be great if this could be addressed.

**Reply by max-sixty** [reply]
This is fixed by https://github.com/PRQL/prql/pull/1516. https://github.com/PRQL/prql/issues/1535 has the broader issue.

Thanks a lot for the question @eitsupi !

**Reply by eitsupi** [reply]
Awesome! Thanks for your prompt work!
